doctr/utils/data.py

```diff
@@ -49,6 +49,7 @@ def download_from_url(
     hash_prefix: Optional[str] = None,
     cache_dir: Optional[str] = None,
     cache_subdir: Optional[str] = None,
+    check_hash: bool = True,
```

Comment:
As mentioned before, we can't forward to change this file :sweat_smile: 
If at some point, we cannot perform the check, then we just don't do it upstream but this function should not be changed. If the name of the file doesn't include any hash and it's not passed, the check won't be done, so that should be enough :ok_hand: